src/System.Data.SqlClient/tests/ManualTests/SQL/CommandCancelTest/CommandCancelTest.cs

```diff
@@ -279,5 +279,65 @@ private static void TimeOutDuringRead(string constr)
                 throw;
             }
         }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
+        public static void CancelDoesNotWait()
+        {
+            const int delaySeconds = 30;
+            const int cancelSeconds = 1;
+
+            using (SqlConnection conn = new SqlConnection(s_connStr))
+            using (var cmd = new SqlCommand($"WAITFOR DELAY '00:00:{delaySeconds:D2}'", conn))
+            {
+                conn.Open();
+
+                Task.Delay(TimeSpan.FromSeconds(cancelSeconds))
+                    .ContinueWith(t => cmd.Cancel());
+
+                DateTime started = DateTime.UtcNow;
+                DateTime ended = default;
```

Comment:
`default` literal not supported in this project.
```suggestion
                DateTime ended = DateTime.UtcNow;
```

Reply:
@Wraith2 can you please commit fix? I don't have permissions to push to your fork.

Reply:
Done, Now isn't the right value, it has to be MInValue.